<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>319</width>
    <height>374</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>319</width>
    <height>374</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>319</width>
    <height>374</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/resources/img/icon.png</normaloff>:/resources/img/icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(90, 51, 0)
</string>
   </property>
   <widget class="QPushButton" name="reButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(200, 161, 110)</string>
    </property>
    <property name="text">
     <string>üîÉ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="trophyButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(200, 161, 110)</string>
    </property>
    <property name="text">
     <string>üèÜ</string>
    </property>
   </widget>
   <widget class="QLabel" name="nickName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color : white</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="time">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>23</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color : white</string>
    </property>
    <property name="text">
     <string>00 : 00</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>-6</x>
      <y>50</y>
      <width>331</width>
      <height>331</height>
     </rect>
    </property>
    <widget class="QWidget" name="page_1">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>9</y>
        <width>313</width>
        <height>313</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="entryButton">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>250</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(200, 161, 110)</string>
       </property>
       <property name="text">
        <string>–ü–æ—á–∞—Ç–∏!</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>180</y>
         <width>191</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color : white</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>20</y>
         <width>101</width>
         <height>111</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>68</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color : white</string>
       </property>
       <property name="text">
        <string>üîí</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>142</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color : white</string>
       </property>
       <property name="text">
        <string>–í–∞—à–µ —ñ–º'—è:</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="font">
         <font>
          <pointsize>58</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="title">
         <string/>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <widget class="QPushButton" name="pushButton_16">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>70</width>
           <height>70</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string/>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(200, 161, 110)</string>
         </property>
         <property name="text">
          <string>1</string>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_17">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>85</y>
           <width>70</width>
           <height>70</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(200, 161, 110)</string>
         </property>
         <property name="text">
          <string>5</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_18">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>160</y>
           <width>70</width>
           <height>70</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(200, 161, 110)</string>
         </property>
         <property name="text">
          <string>9</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_19">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>235</y>
           <width>70</width>
           <height>70</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(200, 161, 110)</string>
         </property>
         <property name="text">
          <string>13</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_20">
         <property name="geometry">
          <rect>
           <x>85</x>
           <y>235</y>
           <width>70</width>
           <height>70</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(200, 161, 110)</string>
         </property>
         <property name="text">
          <string>14</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_21">
         <property name="geometry">
          <rect>
           <x>85</x>
           <y>10</y>
           <width>70</width>
           <height>70</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string/>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(200, 161, 110)</string>
         </property>
         <property name="text">
          <string>2</string>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_22">
         <property name="geometry">
          <rect>
           <x>85</x>
           <y>85</y>
           <width>70</width>
           <height>70</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(200, 161, 110)</string>
         </property>
         <property name="text">
          <string>6</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_23">
         <property name="geometry">
          <rect>
           <x>85</x>
           <y>160</y>
           <width>70</width>
           <height>70</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(200, 161, 110)</string>
         </property>
         <property name="text">
          <string>10</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_24">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>160</y>
           <width>70</width>
           <height>70</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(200, 161, 110)</string>
         </property>
         <property name="text">
          <string>11</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_25">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>235</y>
           <width>70</width>
           <height>70</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(200, 161, 110)</string>
         </property>
         <property name="text">
          <string>15</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_26">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>85</y>
           <width>70</width>
           <height>70</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(200, 161, 110)</string>
         </property>
         <property name="text">
          <string>7</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_27">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>10</y>
           <width>70</width>
           <height>70</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string/>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(200, 161, 110)</string>
         </property>
         <property name="text">
          <string>3</string>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_28">
         <property name="geometry">
          <rect>
           <x>235</x>
           <y>10</y>
           <width>70</width>
           <height>70</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string/>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(200, 161, 110)</string>
         </property>
         <property name="text">
          <string>4</string>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_29">
         <property name="geometry">
          <rect>
           <x>235</x>
           <y>85</y>
           <width>70</width>
           <height>70</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string/>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(200, 161, 110)</string>
         </property>
         <property name="text">
          <string>8</string>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_30">
         <property name="geometry">
          <rect>
           <x>235</x>
           <y>160</y>
           <width>70</width>
           <height>70</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string/>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(200, 161, 110)</string>
         </property>
         <property name="text">
          <string>12</string>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>9</y>
        <width>313</width>
        <height>313</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>58</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="title">
       <string/>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>45</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color : white</string>
         </property>
         <property name="text">
          <string>      –¢–∞–±–ª–∏—Ü—è —Ä–µ–∫–æ—Ä–¥—ñ–≤</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color : white</string>
         </property>
         <property name="text">
          <string>–Ü–º '—è               |                –ß–∞—Å</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="horizontalSpacing">
          <number>170</number>
         </property>
         <property name="verticalSpacing">
          <number>6</number>
         </property>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>9</y>
        <width>313</width>
        <height>313</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>40</y>
         <width>151</width>
         <height>131</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>109</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color : green</string>
       </property>
       <property name="text">
        <string>‚úîÔ∏è</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>180</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color : white</string>
       </property>
       <property name="text">
        <string>–í–∏ –≤–∏–≥—Ä–∞–ª–∏!</string>
       </property>
      </widget>
      <widget class="QPushButton" name="okButton">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>240</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(200, 161, 110)</string>
       </property>
       <property name="text">
        <string>–î–æ–±—Ä–µ</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
